src/core/services/google-maps-types.ts

```diff
@@ -356,3 +357,26 @@ export interface KmlMouseEvent extends MouseEvent {
   featureData: KmlFeatureData;
   pixelOffset: Size;
 }
+
+export interface Data extends MVCObject {
+  constructor(options?: DataOptions): void;
+  addGeoJson(geoJson: Object, options?: GeoJsonOptions): Feature[];
+  remove(feature: Feature): void;
+  setMap(map: GoogleMap): void;
+}
+
+export interface Feature extends MVCObject {
```

Comment:
the properties of Feature are missing here: https://developers.google.com/maps/documentation/javascript/reference#Data.Feature